index.html

```diff
@@ -0,0 +1,149 @@
+        />
+        <div class="display-non-desktop disktop-nav">
+          <div class="close-btn">&#x2715;</div>
+          <a href="./index.html">
+            <img
+              src="./imges/icons/main-logo.svg"
+              alt="header-icon"
+              class="logo"
+            />
+          </a>
+          <ul>
+            <li>
+              <a href="about.html"> <span>About</span></a>
+            </li>
+            <li><a href="./index.html">program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponser</a></li>
+            <li><a href="#">News</a></li>
+            <li>
+              <h4 class="box">camping</h4>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <div class="cover-cont">
+        <h1>
+          "hello! sharing world!" <br />
+          <span>
+            creatve commons <br />
+            global summit <br />
+            2021
+          </span>
+        </h1>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet
+          recusandae quia et? Ut cumque ullam magni Lorem ipsum dolor sit amet
+          consectetur adipisicing elit. Amet recusandae quia et? Ut cumque ullam
+          magni
+        </p>
+        <h3>2021.10.15</h3>
+        <h6>@ Lorem ipsum dolor sit amet consectetur</h6>
+      </div>
+    </section>
+    <section class="programs">
+      <h2>main programs</h2>
+      <hr class="main-program-line" />
+      <ul>
+        <li>
+          <img
+            src="./imges/icons/progrem-text-icon.svg"
+            alt=""
+            class="program-logo"
+          />
+          <h3>lecture</h3>
+          <p>
+            A lecture is an oral presentation intended to present information or
+            teach people
+          </p>
+        </li>
+        <li>
+          <img src="./imges/icons/ex.svg" alt="" class="program-logo" />
+          <h3>Exhibition</h3>
+          <p>
+            An exhibition, in the most general sense, is an organized
+            presentation and display
+          </p>
+        </li>
+        <li>
+          <img
+            src="./imges/icons/progrem-text-icon.svg"
+            alt=""
+            class="program-logo"
+          />
+          <h3>form</h3>
+          <p>
+            An exhibition, in the most general sense, is an organized
+            presentation and display
+          </p>
+        </li>
+        <li>
+          <img src="./imges/icons/workshop.svg" alt="" class="program-logo" />
+          <h3>workshop</h3>
+          <p>
+            a meeting at which a group of people engage in intensive discussion.
+          </p>
+        </li>
+        <li>
+          <img src="./imges/icons/sun.svg" alt="" class="program-logo" />
+          <h3>ignit</h3>
+          <p>
+            adjective. Slang for the word ignorant, Compare to: Silly, Retarted
+          </p>
+        </li>
+      </ul>
+      <h4><a>join global summit 2021</a></h4>
+    </section>
```

Comment:
hello Rose 
it's very hard to make it exactly the same and I can't find icons that can work properly but I will do my best
and what I understand from the project requires that I customize it as I want

Reply:
Hello @yahyaelganyni1 
Thanks for reaching out. For the icons, I should have made it an optional requirement since you can customize it to however you want. Apologies for the inconvenience caused.